src/KiteUtils.jl

```diff
@@ -48,7 +48,7 @@ import Base.length
 import ReferenceFrameRotations as RFR
 export demo_state, demo_syslog, demo_log, load_log, save_log, export_log, import_log # functions for logging
 export log!, syslog, length, euler2rot, menu
-export demo_state_4p, demo_state_4p_3lines, initial_kite_ref_frame       # functions for four point and three line kite
+export demo_state_4p, initial_kite_ref_frame       # functions for four point and three line kite
```

Comment:
Is the comment correct?

Reply:
You did not reply my question. The comment in the new line refers to a three line kite.